config/routes.rb

```diff
@@ -118,7 +118,7 @@
 
     resource :legislation, only: %i[show update]
 
-    scope module: :planning_application do
+    scope module: :planning_applications do
```

Comment:
Do we need to worry about any existing links that this might break? (probably not)

Reply:
The `:module` option doesn't affect either the url generated or the name of the url helper method, it just controls the namespace of the controller so unless there's somewhere we're using `url_for` with `:controller` and `:action` options then it should be fine (I search for `"planning_application/` across the project so should've picked it up anyway).